examples/docs/aincraft/index.html

```diff
@@ -27,19 +27,22 @@
 
     <a-sky id="background" src="#skyTexture" theta-length="90" radius="30"></a-sky>
 
-    <!-- Hands -->
-    <a-entity id="teleHand"
-              hand-controls="hand: left"
-              blink-controls="collisionEntities: [mixin='voxel'], #ground"></a-entity>
-    <a-entity id="blockHand"
-              hand-controls="hand: right"
-              laser-controls
-              intersection-spawn="event: click; mixin: voxel"></a-entity>
-
-    <!-- Camera -->
-    <a-camera>
-      <a-cursor intersection-spawn="event: click; mixin: voxel"></a-cursor>
-    </a-camera>
+    <a-entity id="player">
+      <!-- Hands -->
+      <a-entity id="teleHand"
+                hand-controls="hand: left"
+                blink-controls="collisionEntities: [mixin='voxel'], #ground"></a-entity>
+      <a-entity id="blockHand"
+                hand-controls="hand: right"
+                laser-controls
+                raycaster="near: 0.5"
+                intersection-spawn="event: click; mixin: voxel"></a-entity>
 
+      <!-- Camera -->
+      <a-camera>
+        <!-- Uncomment the line below to place blocks on desktop or mobile -->
```

Comment:
This needs to be commented out?

Reply:
Without that commented out, blocks were getting placed near the controller instead of a distance. This is on the Meta Quest 2 browser.